pkg/registry/admissionregistration/rest/storage_apiserver.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright 2016 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos - file modified.
```

Comment:
Please consult with Xiaoning how we should deal with the copyright of those auto generated files.

Reply:
just talked in scalability meeting, that is OK to add arktos copyrights.

Reply:
Added cherrypick comments for all added/updated files